pkg/kudoctl/cmd/generate/maintainer.go

```diff
@@ -23,10 +23,10 @@ func AddMaintainer(fs afero.Fs, path string, m *kudoapi.Maintainer) error {
 
 // MaintainerList provides a list of operator maintainers
 func MaintainerList(fs afero.Fs, path string) ([]*kudoapi.Maintainer, error) {
-	p, err := reader.ReadDir(fs, path)
+	p, err := reader.FromDir(fs, path)
```

Comment:
```suggestion
	p, err := reader.PackageFilesFromDir(fs, path)
```

I wouldn't mind having a longer name for this func. I know you didn't change it, but it took me a bit to figure out what we actually get from the `FromDir` func...

Reply:
Oh yeah, I wanted to change that too 👍